haystack_experimental/utils/jinja_chat_extension.py

```diff
@@ -0,0 +1,203 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
```

Comment:
I think we are missing an `__init__.py` file in this folder

Reply:
Yes.
I would add an empty `__init__.py` file because I don't want the ChatMessage extension to be easily accessible/usable outside the `ChatPromptBuilder`